packages/app/blockissuer/ratesetter/utils/issuerqueue.go

```diff
@@ -10,20 +10,26 @@ import (
 	"github.com/iotaledger/goshimmer/packages/protocol/models"
 )
 
+const (
+	// MaxLocalQueueSize is the maximum local (containing the block to be issued) queue size in number of blocks.
+	MaxLocalQueueSize = 20
+)
+
 // region IssuerQueue /////////////////////////////////////////////////////////////////////////////////////////////
 
 // IssuerQueue keeps the submitted blocks of an issuer.
 type IssuerQueue struct {
```

Comment:
IssuerQueue is not used anymore in the ratesetters, so remove the whole file.

Reply:
done